react/components/ProductList/Product.tsx

```diff
@@ -32,6 +32,11 @@ const Product: FC<Props> = ({ product }) => {
   } = product
   const handles = useCssHandles(CSS_HANDLES)
   const showMeasurementUnit = unitMultiplier !== 1 || measurementUnit !== 'un'
+  const productSubtotal = useMemo(() => price * quantity * unitMultiplier, [
+    price,
+    quantity,
+    unitMultiplier,
+  ])
```

Comment:
It's a cheap computation, no need for useMemo
```suggestion
  const productSubtotal = price * quantity * unitMultiplier
```